### ISSUE TITLE
add documentation re: where to put {valtools} in DESCRIPTION

### ISSUE THREAD

**Issue by mariev** [phuse-org/valtools#143]
Although we are using {valtools} primarily in the validation report the package needs to be added to the import field due to use of roxygen2 + dynamic labels, when part of package dev. 

Routine use of packages for vignettes only are added to the suggests field which will cause devtools::check.

Makes sense if ppl are aware of the extensibility {roxygen2} --> {valtools}, but may be a new bit of knowledge for ppl not digging in so far.

**Comment by thebioengineer** [comment]
Probably makes sense to add to "suggests" ? I think then it is still captured and cran checks/etc load suggests AFAIK. could have that as part of `vt_use_validation()` is to add it to the "suggests" when in a package?

**Comment by mariev** [comment]
no, if you put in `suggests` when developing pkg e.g. {scharpbama} it will throw devtools::check warning b/c things like `@editor` and `@editDate` are not defined.

**Comment by thebioengineer** [comment]
Interesting, so it needs to be added to "Imports"?

**Comment by mariev** [comment]
There is probably an incredibly niche use case where the developer does not take advantage of the @editor/@editDate tags (dynamic referencing would fall under this umbrella), which would allow for {valtools} to be placed in Suggests, but def not our default usage.

With the use case illustrated in {valtools} vignette, developers will need to add to their package imports. 

Note that if they choose Depends, there is a *different* message that comes up if the pkg is not also included in the NAMESPACE imports list.
